pages/profiles.html

```diff
@@ -82,15 +82,23 @@
   <li><code>no-swagger</code> to disable swagger</li>
   <li><code>no-liquibase</code> to disable liquibase</li>
 </ul>
-<p>These can be used along with both <code>dev</code> and <code>prod</code> profiles.</p>
+<p>These can be used along with both <code>dev</code> and <code>prod</code> profiles. 
+Please note that <code>no-swagger</code> doesnt have any effect on <code>prod</code> profile as swagger is adisabled by default on that profile.</p>
 <p>The are only used at run time:</p>
 
 <ul>
-    <li>In your IDE, run your main applcation class with <code>spring.profiles.active=dev,no-swagger</code> (please note you need to activate the "dev" profile)</li>
-    <li>With a packaged application: <code>./java -jar jhipster-0.0.1-SNAPSHOT.war --spring.profiles.active=prod,no-liquibase,no-swagger</code></li>
+    <li>In your IDE, run your main applcation class with <code>spring.profiles.active=dev,no-swagger,no-liquibase</code> (please note you need to activate the "dev" or "prod" profile explicitly)</li>
+    <li>With a packaged application: <code>./java -jar jhipster-0.0.1-SNAPSHOT.war --spring.profiles.active=prod,no-liquibase</code></li>
 </ul>
 
-<p>With Gradle (this is not supported with Maven), you can also use those profiles directly:<p>
+<p>With Maven, you can also use those profiles directly:<p>
+
+<ul>
+    <li><code>./mvnw -Pprod "-Drun.profiles=no-liquibase"</code></li>
```

Comment:
Never heard of it, I need to check where this comes from.
Le 25 mars 2016 5:00 PM, "Deepu K Sasidharan" <notifications@github.com> a
écrit :

> In pages/profiles.html
> <https://github.com/jhipster/jhipster.github.io/pull/235#discussion_r57457660>
> :
>
> >  </ul>
> >
> > -<p>With Gradle (this is not supported with Maven), you can also use those profiles directly:<p>
> > +<p>With Maven, you can also use those profiles directly:<p>
> > +
> > +<ul>
> > +    <li><code>./mvnw -Pprod "-Drun.profiles=no-liquibase"</code></li>
>
> Yes I have tested it works
> … <#m_2609607781150647222_>
> On 25 Mar 2016 21:29, "Julien Dubois" <notifications@github.com> wrote:
> In pages/profiles.html <#235 (comment)
> <https://github.com/jhipster/jhipster.github.io/pull/235#discussion_r57443605>>
> : > </ul> > > -<p>With Gradle (this is not supported with Maven), you can
> also use those profiles directly:<p> > +<p>With Maven, you can also use
> those profiles directly:<p> > + > +<ul> > + <li><code>./mvnw -Pprod
> "-Drun.profiles=no-liquibase"</code></li> I didn't know about this
> "run.profiles" key, are you sure about this? — You are receiving this
> because you authored the thread. Reply to this email directly or view it on
> GitHub <
> https://github.com/jhipster/jhipster.github.io/pull/235/files/2dd8a33b139fe92bfe5f53973fba7be487879bd5#r57443605>
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/jhipster.github.io/pull/235/files/2dd8a33b139fe92bfe5f53973fba7be487879bd5#r57457660>
>